nifi-nar-bundles/nifi-mqtt-bundle/nifi-mqtt-processors/src/main/java/org/apache/nifi/processors/mqtt/common/AbstractMQTTProcessor.java

```diff
@@ -307,7 +277,7 @@ public Collection<ValidationResult> customValidate(final ValidationContext valid
         final boolean readerIsSet = validationContext.getProperty(BASE_RECORD_READER).isSet();
         final boolean writerIsSet = validationContext.getProperty(BASE_RECORD_WRITER).isSet();
         if ((readerIsSet && !writerIsSet) || (!readerIsSet && writerIsSet)) {
-            results.add(new ValidationResult.Builder().subject("Reader and Writer").valid(false)
+            results.add(new ValidationResult.Builder().subject("Record Reader and Writer").valid(false)
                     .explanation("both Record Reader and Writer must be set when used.").build());
```

Comment:
Rendered text on the UI looks like "Record Reader and Writer is invalid, because both Record Reader and Writer must be set when used." which is a bit verbose. I'd remove the reference to "Record Reader and Writer" from the explanation to make the description simpler.